analysis/src/SharedTypes.ml

```diff
@@ -425,11 +428,18 @@ module Completable = struct
     | CPObj of contextPath * string
     | CPPipe of contextPath * string
 
-  type typedContext = NamedArg of string
+  type namedArg = {
+    (* What the user has already started writing, if anything *)
+    prefix: string;
+    (* Name of the argument *)
+    name: string;
```

Comment:
Perhaps `label` as in labelled argument, instead of `name`? The thing is `prefix` is also going to be the name of something, in a slightly different sense.